link fails in toolboxmor and feelpp package version mismatch between feelpp and toolboxes/mor

we have this issue at link time in toolboxmor preventing completion of packaging

see https://buildkite.com/feelpp/pkg-feelpp-mor-focal/builds/24#7b5a80d4-2f4e-4434-83db-c2e49d9904ca/1711-4771

```
/usr/bin/clang++-9  -DNDEBUG -O2 -Wdate-time -D_FORTIFY_SOURCE=2 -DNDEBUG -O2 -Wdate-time -D_FORTIFY_SOURCE=2 -ftemplate-depth=1024 -stdlib=libstdc++ -fvisibility-inlines-hidden -pthread -DNDEBUG -O2 -Wdate-time -D_FORTIFY_SOURCE=2 -ftemplate-depth=1024 -stdlib=libstdc++ -fvisibility-inlines-hidden -pthread -DNDEBUG -O3  -Wl,--no-as-needed -lm -lrt CMakeFiles/feelpp_mor_toolbox-opusheat.dir/toolboxmor_main.cpp.o  -o feelpp_mor_toolbox-opusheat  -Wl,-rpath,"/build/feelpp-mor-0.109.0~alpha.1/build-python3.8/toolbox-mor:/usr/lib/x86_64-linux-gnu/openmpi/lib:/usr/lib/x86_64-linux-gnu/hdf5/openmpi/lib:/usr/lib/petscdir/petsc3.12/x86_64-linux-gnu-real/lib:/usr/lib/x86_64-linux-gnu/hdf5/openmpi" libfeelpp_crb_toolbox-mor.so -lfeelpp_toolbox_heat_lib /usr/lib/x86_64-linux-gnu/libfeelpp.so.1.0.0 /usr/lib/x86_64-linux-gnu/libfeelpp_fmt.so.7.1.3 -Wl,--as-needed /usr/lib/x86_64-linux-gnu/libfeelpp_glog.so.0 /usr/lib/x86_64-linux-gnu/libfeelpp_gflags_nothreads.so.2.2.0 /usr/lib/x86_64-linux-gnu/libfeelpp_ginac.so.6.0.1 /usr/lib/x86_64-linux-gnu/libcln.so /usr/lib/x86_64-linux-gnu/libboost_mpi.so /usr/lib/x86_64-linux-gnu/libboost_serialization.so /usr/lib/x86_64-linux-gnu/libboost_filesystem.so /usr/lib/x86_64-linux-gnu/libfeelpp_metis.so.1.0.0 /usr/lib/x86_64-linux-gnu/libfeelpp_nlopt.so.0.9.0 -lm /usr/lib/x86_64-linux-gnu/libfeelpp_mongocxx.so.3.1.1 /usr/lib/x86_64-linux-gnu/libfeelpp_bsoncxx.so.3.1.1 /usr/lib/x86_64-linux-gnu/libfeelpp_kwsys.so.1.0.0 /usr/lib/x86_64-linux-gnu/libfeelpp_fmilib.so.1.0.0 /usr/lib/x86_64-linux-gnu/libfeelpp_fmiimport.so.1 /usr/lib/x86_64-linux-gnu/libfeelpp_fmixml.so.1 /usr/lib/x86_64-linux-gnu/libfeelpp_expat.so.2.1.0 /usr/lib/x86_64-linux-gnu/libfeelpp_fmizip.so.1 /usr/lib/x86_64-linux-gnu/libfeelpp_minizip.so.1 /usr/lib/x86_64-linux-gnu/libfeelpp_z.so.1.2.6 /usr/lib/x86_64-linux-gnu/libfeelpp_fmicapi.so.1 /usr/lib/x86_64-linux-gnu/libfeelpp_jmutils.so.1 -ldl /usr/lib/x86_64-linux-gnu/libfeelpp_c99snprintf.so.1 -lpthread /usr/lib/x86_64-linux-gnu/libboost_system.so /usr/lib/x86_64-linux-gnu/libboost_thread.so /usr/lib/x86_64-linux-gnu/libboost_chrono.so /usr/lib/x86_64-linux-gnu/libboost_date_time.so /usr/lib/x86_64-linux-gnu/libboost_atomic.so /usr/lib/x86_64-linux-gnu/libssl.so /usr/lib/x86_64-linux-gnu/libcrypto.so /usr/lib/x86_64-linux-gnu/libfeelpp_parmmg.so /usr/lib/x86_64-linux-gnu/libfeelpp_mmg.so.5.5.2 /usr/lib/x86_64-linux-gnu/libmetis.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_cxx.so /usr/lib/x86_64-linux-gnu/libfeelpp_mmgs.so.5.5.2 /usr/lib/x86_64-linux-gnu/libfeelpp_mmg3d.so.5.5.2 -pthread /usr/lib/x86_64-linux-gnu/libscotch.so /usr/lib/x86_64-linux-gnu/libscotcherrexit.so -lz -lm -lrt /usr/lib/x86_64-linux-gnu/libfeelpp_cpr.so.1.5.1 /usr/lib/x86_64-linux-gnu/libcurl.so /usr/lib/x86_64-linux-gnu/libcurl.so /usr/lib/x86_64-linux-gnu/hdf5/openmpi/lib/libhdf5.so /usr/lib/x86_64-linux-gnu/libsz.so /usr/lib/x86_64-linux-gnu/libz.so /usr/lib/x86_64-linux-gnu/libboost_date_time.so /usr/lib/x86_64-linux-gnu/libboost_filesystem.so /usr/lib/x86_64-linux-gnu/libboost_system.so /usr/lib/x86_64-linux-gnu/libboost_program_options.so /usr/lib/x86_64-linux-gnu/libboost_unit_test_framework.so /usr/lib/x86_64-linux-gnu/libboost_mpi.so /usr/lib/x86_64-linux-gnu/libboost_serialization.so /usr/lib/x86_64-linux-gnu/libboost_iostreams.so /usr/lib/x86_64-linux-gnu/libboost_regex.so /usr/lib/x86_64-linux-gnu/libdl.so /usr/lib/x86_64-linux-gnu/libfftw3.so /usr/lib/x86_64-linux-gnu/libfftw3_mpi.so /usr/lib/x86_64-linux-gnu/libfftw3f.so /usr/lib/x86_64-linux-gnu/libfftw3l.so /usr/lib/x86_64-linux-gnu/libann.so /usr/lib/x86_64-linux-gnu/libglpk.so /usr/lib/petscdir/petsc3.12/x86_64-linux-gnu-real/lib/libpetsc_real.so /usr/lib/x86_64-linux-gnu/libHYPRE_core.so /usr/lib/x86_64-linux-gnu/libdmumps.so /usr/lib/x86_64-linux-gnu/libzmumps.so /usr/lib/x86_64-linux-gnu/libsmumps.so /usr/lib/x86_64-linux-gnu/libcmumps.so /usr/lib/x86_64-linux-gnu/libmumps_common.so /usr/lib/x86_64-linux-gnu/libpord.so /usr/lib/x86_64-linux-gnu/libscalapack-openmpi.so /usr/lib/x86_64-linux-gnu/libumfpack.so /usr/lib/x86_64-linux-gnu/libamd.so /usr/lib/x86_64-linux-gnu/libcholmod.so /usr/lib/x86_64-linux-gnu/libklu.so /usr/lib/x86_64-linux-gnu/libsuperlu.so /usr/lib/x86_64-linux-gnu/libsuperlu_dist.so /usr/lib/x86_64-linux-gnu/liblapack.so /usr/lib/x86_64-linux-gnu/libblas.so /usr/lib/x86_64-linux-gnu/libptesmumps.so /usr/lib/x86_64-linux-gnu/libptscotch.so /usr/lib/x86_64-linux-gnu/libptscotcherr.so /usr/lib/x86_64-linux-gnu/hdf5/openmpi/libhdf5.so /usr/lib/x86_64-linux-gnu/libOpenCL.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_usempif08.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_usempi_ignore_tkr.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_mpifh.so /usr/lib/gcc/x86_64-linux-gnu/9/libgfortran.so -lgcc_s /usr/lib/x86_64-linux-gnu/libpthread.so /usr/lib/gcc/x86_64-linux-gnu/9/libquadmath.so /usr/lib/gcc/x86_64-linux-gnu/9/libstdc++.so /usr/lib/x86_64-linux-gnu/libdl.so /usr/lib/x86_64-linux-gnu/libfftw3.so /usr/lib/x86_64-linux-gnu/libfftw3_mpi.so /usr/lib/x86_64-linux-gnu/libfftw3f.so /usr/lib/x86_64-linux-gnu/libfftw3l.so /usr/lib/x86_64-linux-gnu/libann.so /usr/lib/x86_64-linux-gnu/libglpk.so /usr/lib/petscdir/petsc3.12/x86_64-linux-gnu-real/lib/libpetsc_real.so /usr/lib/x86_64-linux-gnu/libHYPRE_core.so /usr/lib/x86_64-linux-gnu/libdmumps.so /usr/lib/x86_64-linux-gnu/libzmumps.so /usr/lib/x86_64-linux-gnu/libsmumps.so /usr/lib/x86_64-linux-gnu/libcmumps.so /usr/lib/x86_64-linux-gnu/libmumps_common.so /usr/lib/x86_64-linux-gnu/libpord.so /usr/lib/x86_64-linux-gnu/libscalapack-openmpi.so /usr/lib/x86_64-linux-gnu/libumfpack.so /usr/lib/x86_64-linux-gnu/libamd.so /usr/lib/x86_64-linux-gnu/libcholmod.so /usr/lib/x86_64-linux-gnu/libklu.so /usr/lib/x86_64-linux-gnu/libsuperlu.so /usr/lib/x86_64-linux-gnu/libsuperlu_dist.so /usr/lib/x86_64-linux-gnu/liblapack.so /usr/lib/x86_64-linux-gnu/libblas.so /usr/lib/x86_64-linux-gnu/libptesmumps.so /usr/lib/x86_64-linux-gnu/libptscotch.so /usr/lib/x86_64-linux-gnu/libptscotcherr.so /usr/lib/x86_64-linux-gnu/hdf5/openmpi/libhdf5.so /usr/lib/x86_64-linux-gnu/libOpenCL.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_usempif08.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_usempi_ignore_tkr.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_mpifh.so /usr/lib/gcc/x86_64-linux-gnu/9/libgfortran.so -lgcc_s /usr/lib/x86_64-linux-gnu/libpthread.so /usr/lib/gcc/x86_64-linux-gnu/9/libquadmath.so /usr/lib/gcc/x86_64-linux-gnu/9/libstdc++.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi.so -lm /usr/lib/x86_64-linux-gnu/libslepc.so /usr/lib/x86_64-linux-gnu/libgsl.so /usr/lib/x86_64-linux-gnu/libgslcblas.so /usr/lib/x86_64-linux-gnu/libgl2ps.so /usr/lib/x86_64-linux-gnu/libGL.so /usr/lib/x86_64-linux-gnu/libgmsh.so /usr/lib/x86_64-linux-gnu/libxml2.so
--
  | /usr/bin/ld: CMakeFiles/feelpp_mor_toolbox-opusheat.dir/toolboxmor_main.cpp.o: undefined reference to symbol '_ZN4Feel10FeelModels4HeatINS_7SimplexILt2ELt1ELt2EEENS_8LagrangeILt1ENS_6ScalarENS_10ContinuousENS_14PointSetFeketeELt0EEEE5solveEv'
  | /usr/bin/ld: /usr/lib/x86_64-linux-gnu/libfeelpp_toolbox_heat_lib_2dP1G1.so.1.0: error adding symbols: DSO missing from command line
  | clang: error: linker command failed with exit code 1 (use -v to see invocation)
  | make[4]: *** [toolbox-mor/CMakeFiles/feelpp_mor_toolbox-opusheat.dir/build.make:225: toolbox-mor/feelpp_mor_toolbox-opusheat] Error 1
  | make[4]: Leaving directory '/build/feelpp-mor-0.109.0~alpha.1/build-python3.8'
  | make[3]: *** [CMakeFiles/Makefile2:1313: toolbox-mor/CMakeFiles/feelpp_mor_toolbox-opusheat.dir/all] Error 2
  | make[3]: *** Waiting for unfinished jobs....
```

this can be reproduced on buster:
```
/usr/bin/ld: CMakeFiles/feelpp_mor_toolbox-opusheat.dir/toolboxmor_main.cpp.o: undefined reference to symbol '_ZN4Feel10FeelModels4HeatINS_7SimplexILt2ELt1ELt2EEENS_8LagrangeILt1ENS_6ScalarENS_10ContinuousENS_14PointSetFeketeELt0EEEE5solveEv'
  | /usr/bin/ld: //usr/lib/x86_64-linux-gnu/libfeelpp_toolbox_heat_lib_2dP1G1.so.1.0: error adding symbols: DSO missing from command line
```

--- a/mor/thermoelectric/CMakeLists.txt
+++ b/mor/thermoelectric/CMakeLists.txt
@@ -101,7 +101,7 @@ feelpp_add_application( biotsavart-app
                         LINK_LIBRARIES ${CRB_LIB_THERMOELECTRIC_NONLINEAR} INSTALL )
 feelpp_add_application( biotsavart-electric-alpha-app
                         SRCS electric-alpha.hpp biotsavart-electric-alpha.hpp biotsavart-electric-alpha_main.cpp
-                        LINK_LIBRARIES ${CRB_LIB_BIOTSAVART_ELECTRIC_ALPHA} feelpp_toolbox_electric_lib feelpp_modelcore )
+                        LINK_LIBRARIES ${CRB_LIB_BIOTSAVART_ELECTRIC_ALPHA} Feelpp::feelpp_toolbox_electric_lib Feelpp::feelpp_modelcore )
 
 # feelpp_add_application( test SRCS test.cpp )
 

--- a/mor/toolbox-mor/CMakeLists.txt
+++ b/mor/toolbox-mor/CMakeLists.txt
@@ -4,7 +4,7 @@ crb_add_library( toolbox-mor SRCS toolboxmor.cpp EXEC CRB_LIB_TOOLBOXMOR )
 set(FEELPP_CRB_toolboxmor_LIBRARIES ${CRB_LIB_TOOLBOXMOR} PARENT_SCOPE)
 set(FEELPP_CRB_LIBRARIES ${FEELPP_CRB_LIBRARIES} FEELPP_CRB_toolboxmor_LIBRARIES PARENT_SCOPE)
 
-feelpp_add_application(toolbox-opusheat SRCS toolboxmor_main.cpp LINK_LIBRARIES ${CRB_LIB_TOOLBOXMOR} feelpp_toolbox_heat_lib)
+feelpp_add_application(toolbox-opusheat SRCS toolboxmor_main.cpp LINK_LIBRARIES ${CRB_LIB_TOOLBOXMOR} Feelpp::feelpp_toolbox_heat_lib)
 target_include_directories(feelpp_mor_toolbox-opusheat PUBLIC ${CMAKE_SOURCE_DIR}/toolboxes ${CMAKE_BINARY_DIR}/toolboxes)
 if (EXISTS ${FEELPP_INSTALL_DIR}/include/feelpp/toolboxes)
   MESSAGE(STATUS "Loading toolboxes hearders from ${FEELPP_INSTALL_DIR}/include/feelpp/toolboxes")

